src/isyntax/isyntax.c

```diff
@@ -1817,95 +1816,102 @@ u32* isyntax_load_tile(isyntax_t* isyntax, isyntax_image_t* wsi, i32 scale, i32
+			for (i32 y = 0; y < block_height; ++y) {
+				memcpy(dest, source, row_copy_size);
+				dest += dest_stride;
+				source += idwt_stride;
 			}
-			// Blit top right child LL block
-			{
-				icoeff_t* dest = child_top_right->color_channels[color].coeff_ll;
-				icoeff_t* source = idwt + (first_valid_pixel * idwt_stride) + first_valid_pixel + block_width;
-				for (i32 y = 0; y < block_height; ++y) {
-					memcpy(dest, source, row_copy_size);
-					dest += dest_stride;
-					source += idwt_stride;
-				}
+		}
+		// Blit top right child LL block
+		{
+			icoeff_t* dest = child_top_right->color_channels[color].coeff_ll;
+			icoeff_t* source = idwt + (first_valid_pixel * idwt_stride) + first_valid_pixel + block_width;
+			for (i32 y = 0; y < block_height; ++y) {
+				memcpy(dest, source, row_copy_size);
+				dest += dest_stride;
+				source += idwt_stride;
 			}
-			// Blit bottom left child LL block
-			{
-				icoeff_t* dest = child_bottom_left->color_channels[color].coeff_ll;
-				icoeff_t* source = idwt + ((first_valid_pixel + block_height) * idwt_stride) + first_valid_pixel;
-				for (i32 y = 0; y < block_height; ++y) {
-					memcpy(dest, source, row_copy_size);
-					dest += dest_stride;
-					source += idwt_stride;
-				}
+		}
+		// Blit bottom left child LL block
+		{
+			icoeff_t* dest = child_bottom_left->color_channels[color].coeff_ll;
+			icoeff_t* source = idwt + ((first_valid_pixel + block_height) * idwt_stride) + first_valid_pixel;
+			for (i32 y = 0; y < block_height; ++y) {
+				memcpy(dest, source, row_copy_size);
+				dest += dest_stride;
+				source += idwt_stride;
 			}
-			// Blit bottom right child LL block
-			{
-				icoeff_t* dest = child_bottom_right->color_channels[color].coeff_ll;
-				icoeff_t* source = idwt + ((first_valid_pixel + block_height) * idwt_stride) + first_valid_pixel + block_width;
-				for (i32 y = 0; y < block_height; ++y) {
-					memcpy(dest, source, row_copy_size);
-					dest += dest_stride;
-					source += idwt_stride;
-				}
+		}
+		// Blit bottom right child LL block
+		{
+			icoeff_t* dest = child_bottom_right->color_channels[color].coeff_ll;
+			icoeff_t* source = idwt + ((first_valid_pixel + block_height) * idwt_stride) + first_valid_pixel + block_width;
+			for (i32 y = 0; y < block_height; ++y) {
+				memcpy(dest, source, row_copy_size);
+				dest += dest_stride;
+				source += idwt_stride;
 			}
+		}
 
-			// After the last color channel, we can report that the children now have their LL blocks available.
-			if (color == 2) {
-				child_top_left->has_ll = true;
-				child_top_right->has_ll = true;
-				child_bottom_left->has_ll = true;
-				child_bottom_right->has_ll = true;
-
-				// Even if the parent tile has invalid edges around the outside, its child LL blocks will still have valid edges on the inside.
-				child_top_left->ll_invalid_edges = invalid_edges & ~(ISYNTAX_ADJ_TILE_CENTER_RIGHT | ISYNTAX_ADJ_TILE_BOTTOM_RIGHT | ISYNTAX_ADJ_TILE_BOTTOM_CENTER);
```

Comment:
After deleting this the `isyntax_tile_t.ll_invalid_edges` field has no uses anymore. Can it be deleted?